backend/authStrategies/index.ts

```
@@ -16,3 +17,17 @@ export function addAdminIfNone(user: HydratedDocument<IUser>) {
     }
   })
 }
+
+export abstract class AuthenticationStrategy<Strategy extends passport.Strategy, Options extends any> {
```

Biome

[error] Constraining a type parameter to any or unknown is useless.

All types are subtypes of any and unknown.
Safe fix: Remove the constraint.


(lint/complexity/noUselessTypeConstraint)